unit_tests/TestUtils.h

```diff
@@ -160,7 +159,7 @@ struct TestParams {
     }
 };
 
-bool TestParams::skipSerialTests = true;
+bool TestParams::skipSerialTests = false;
```

Comment:
This change doesn't belong in this PR and it also makes the `--run-serial` flag useless, since the tests will never be skipped. I believe it was decided to remove this entirely anyway, so instead of changing the default values, all of this should just be removed (possibly not in this PR).